src/main/java/com/wcc/platform/controller/AboutController.java

```diff
@@ -48,8 +50,12 @@ public ResponseEntity<TeamPage> getTeamPage() {
   @GetMapping("/collaborators")
   @Operation(summary = "API to retrieve information about collaborators")
   @ResponseStatus(HttpStatus.OK)
-  public ResponseEntity<CollaboratorPage> getCollaboratorPage() {
-    return ResponseEntity.ok(cmsService.getCollaborator());
+  public ResponseEntity<CollaboratorPage> getCollaboratorPage(
+      @RequestParam(defaultValue = "1") final int currentPage,
+      @Min(value = 1, message = "Page size must be greater than zero")
+          @RequestParam(defaultValue = "10")
+          final int pageSize) {
+    return ResponseEntity.ok(cmsService.getCollaborator(currentPage, pageSize));
```

Comment:
I think would be better to use `Pagination` object instead of parameters

Reply:
just keep like this does not make sense to send this object, because the other attributes should be response.